pkg/probe/bgp_neighbor_routes_test.go

```diff
@@ -0,0 +1,57 @@
+package probe
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+func TestBGPNeighborPathsIPv4(t *testing.T) {
```

Comment:
Add:
```go
	if err := config.Init(); err != nil {
		t.Fatalf("config.Init failed: %+v", err)
	}
```

Reply:
I got this working with your fix :) I also added a test "config" file to avoid getting errors when testing :)